benchmarks/benchmarks.rb

```diff
@@ -216,15 +224,15 @@ def positional_splat_keyword_and_double_splat_args(a, *args, b:, **kwargs)
     # Run once with each set of arguments to memoize the result values, so our
     # benchmark only tests memoized retrieval time.
     ARGUMENTS.each do |a, b|
-      instance.positional_splat_keyword_and_double_splat_args(a, b, a: a, b: b)
+      instance.positional_splat_keyword_and_double_splat_args(a, b, b: b, a: a)
```

Comment:
I switched the order of these args because `b: b` is the keyword arg and `a: a` is `**kwargs` and I'm worried that `a: a, b: b` might put them both in through `**kwargs` and bypass the keyword arg.